lib/init.coffee

```diff
@@ -252,14 +252,17 @@ module.exports =
           messages = @parseSchemaMessages(textEditor, output)
         if messages.length
           for message in messages
-            message.type = 'Error'
-            message.text = message.text + ' (' + schemaUrl + ')'
-            message.filePath = textEditor.getPath()
+            message.severity = 'error'
+            message.excerpt = message.excerpt + ' (' + schemaUrl + ')'
```

Comment:
Maybe some heuristic: if `schemaUrl` contains `://` then put it into `message.url`, otherwise keep it where it is?